cpp/benchmarks/io/text/multibyte_split.cpp

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+#include "io/text/bgzip_utils.hpp"
```

Comment:
This is the only instance I can find in libcudf of a benchmark including a header from the `src` directory. I think this is not a great pattern and suggests that the header should be public, but `detail`.

I'd move to `<cudf/io/text/bgzip_utils.hpp>`.